hashes.py

```diff
@@ -0,0 +1,128 @@
+"""
+Artem
+
+The requirements for a Hash Function:
+
+1. A hash function must be consistent(deterministic). Every time it receives the sam input (like "aqua"), it must return the same output (like 4). If it's not deterministic, it's no a hash function. Same input will always return the same output. 
+
+2. Different input data should return different numbers to minimize collisions. 
+
+3. A hash function must return numbers that are within a specific range. 
+
+Hashing: blockchain, saving passwords in the BE, SHA-256 (256 bits limit), SHA-512 (used in class), etc. 
+SHA: Secure Hashing Algo
+
+"""
+# Magic Function: 
+# takes a word --> returns the index fo where that word is located in some array of
+# book --> 1
+# egypt --> 4
+# fake_word --> None
+# this fn is O(1)
+
+# HASH FUNCTIONS
+# Any string input ---> A specific number (within some range)
+# MOST IMPORTANT!
+# This fn must be deterministic
+# Same input will alwasy return the same output
+# Non-reversible
+# Avoid collisions
+
+hash_table = [None] * 8 # 8 slots, all initialized to None
```

Comment:
Just think it's cool that you chose the function application method as opposed to the class based method for implementing the HT. When you find yourself in a technical interview, always ask if you can do your data structures like this, because sometimes it's just easier to think about applying functions to data, versus using a method on an object to change the object (smells recursive, no?).